.Dockerignore

```diff
@@ -0,0 +1,21 @@
+### A list of files and file patterns to be excluded from the Dockerfile
```

Comment:
I just tested it, and at least for me this file needs to be `.dockerignore` (all lower case)

Reply:
yes, you are right. I thought I renamed it, but I guess I did not. Should be fixed now.